### DISCUSSION TITLE
Customizing theme-classic's Navbar/Layout returns an error

### DISCUSSION THREAD

**Discussion by sradu** [facebook/docusaurus#7792] [General]
Hi!

I'm trying to add a horizontal menu under the navbar for certain docs categories. Since this is not possible via docusaurus right now I tried to overwrite the NavBar/Layout/index.tsx theme file to add it.

Unfortunately I'm getting this error:
```
Hook useNavbarMobileSidebar is called outside the <NavbarMobileSidebarProvider>.
```

Any suggestions on how to make it happy?

Thank you!

**Comment by slorber** [comment]
We can't help without seeing a runnable sandbox
